core/src/linuxforhealth/healthos/core/app/__init__.py

```diff
@@ -144,3 +148,20 @@ def configure_inbound_endpoints(
         logger.info(
             f"Adding Inbound RestEndpoint Connector to {APP_BASE_URL}/{r.prefix}"
         )
+
+
+async def close_connectors():
+    """Closes down registered connectors"""
+
+    logger.info("Shutting down HealthOS Core Connectors")
+
+    for k in get_kafka_consumer_connectors():
+        logger.info(f"Stopping Kafka Connectors")
+        await k.stop()
+
+    for n in get_jetstream_client():
+        logger.info(f"Stopping NATS Jetstream Connectors")
+        await n._nc.close()
```

Comment:
@ccorley - I'm accessing the "protected" attribute here for the "first pass". Do you have suggestions on a more appropriate "shutdown" process?

Reply:
client.close() is the correct way to shut down.  In the past we've used 1 NATS client to manage connections to multiple NATS servers, so there was only 1 shutdown call, which is a difference.